src/sheet_metal_conus.py

```diff
@@ -309,9 +447,8 @@ def beVerbose(self, dictCone, unitFactor, parent):
                         'd': 'M %f,%f L %f,%f' %(-(5+cut_dia/2*unitFactor),0, -(5+base_dia/2*unitFactor),0 )}
         line = inkex.etree.SubElement(parent, inkex.addNS('path','svg'), line_attribs)
         simpletransform.applyTransformToNode(frustrum_repos, line)
-        line_attribs = {'style': simplestyle.formatStyle(line_style),
-                        'd': 'M %f,%f L %f,%f' %(-base_dia/2*unitFactor,0, -base_dia/2*unitFactor,cone_height*unitFactor-5 )}
-        line = inkex.etree.SubElement(parent, inkex.addNS('path','svg'), line_attribs)
+        #
```

Comment:
This is the vertical cone height dimension.
I elected to not draw the two cone diameter dimensions with arrows. Let me know what you think about that.